docs/ou3-proof-research-state.md

```
@@ -93,42 +93,425 @@ but is not the limiter.
+
+Iterating the joint word map `P <- max over law-consistent cells of
+word_map(cell, P)` from the declared `P0` converges in 81 iterations:
+
+| | v | p | S | a_w |
+| --- | --- | --- | --- | --- |
+| `P*` (variance) | 1578.4 | 10879 | 25699 | 35.80 |
+| as std | 39.7 m/s | 104 m | 160 m.s | 5.98 m/s^2 |
+
+`P*` is **invariant under every law-consistent cell** (verified, not argued), and
+
+    worst delta over all law-consistent cells = 1.26e-8
```

markdownlint-cli2 (0.23.2)

[warning] Code block style
Expected: fenced; Actual: indented

(MD046, code-block-style)